app/templates/rentals/show.hbs

```diff
@@ -0,0 +1 @@
+{{rental-listing rental=rental}}
```

Comment:
fyi, @kidgodzilla is working on providing markup/style for the details page.  Just showing the listing component is a great placeholder though.